core/services/ocrcommon/block_translator.go

```diff
@@ -21,8 +21,6 @@ func NewBlockTranslator(cfg Config, client evmclient.Client, lggr logger.Logger)
 	switch cfg.ChainType() {
 	case config.ChainArbitrum:
 		return NewArbitrumBlockTranslator(client, lggr)
-	case config.ChainXDai, config.ChainMetis, config.ChainOptimismBedrock:
-		fallthrough
```

Comment:
The linter insists on being exhaustive

Reply:
Yea this switch here confused me as it wasn't exhaustive before and I wanted to align both